subgraphs/aave-v2-forks/protocols/radiant/src/constants.ts

```diff
@@ -16,7 +16,7 @@ export const RWETH_ADDRESS = "0x15b53d277af860f51c3e6843f8075007026bbb3a";
 
 export namespace Protocol {
   export const NAME = "Radiant Captital";
-  export const SLUG = "radiant-capital";
+  export const SLUG = "radiant";
```

Comment:
why not `radiant-capital`? https://www.coingecko.com/en/coins/radiant-capital

Reply:
The hosted service deployment was just `radiant-arbitrum`, so dropped the "capital" to make it consistent with that 😅 

It only takes a few days to sync, so can probably deploy it into a new one with the correct name.